js/utility/bc-google-analytics.js

```diff
@@ -0,0 +1,68 @@
+// Requires Gtag from Google Analytics, requires includes polyfill
+
+namespacer('bcpl.utility');
+
+bcpl.utility.googleAnalytics = (() => {
+	let gtag;
+	let validHostNames = ['www.bcpl.info', 'bcpl.info', 'catalog.bcpl.lib.md.us', 'www.catalog.bcpl.lib.md.us'];
```

Comment:
@martypowell and I discussed in IM, but I want to record somewhere that this hard-coded list of domain names is a maintenance concern.